mobc-redis/src/lib.rs

```diff
@@ -27,12 +32,12 @@ impl Manager for RedisConnectionManager {
     }
 
     async fn check(&self, mut conn: Self::Connection) -> Result<Self::Connection, Self::Error> {
-        let pong: String = redis::cmd("PING").query_async(&mut conn).await?;
-        if pong.as_str() != "PONG" {
+        let ping = self.ping_count.fetch_add(1, Ordering::SeqCst);
```

Comment:
```suggestion
        let ping = self.ping_count.fetch_add(1, Ordering::Relaxed);
```

no need for `SeqCst` here